source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ACRHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) img;
```

Comment:
It's true that it won't make any copies of the Image resource. It will just make some copies of shared_ptr objects, which will entail some extra reference counting operations. It's not very expensive, but it's unnecessary.

---
In reply to: [122552703](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122552703) [](ancestors = 122552703,122550418)